core/src/main/kotlin/com/justai/jaicf/channel/ConsoleChannel.kt

```diff
@@ -15,41 +16,51 @@ import java.util.*
  * @see [BotApi]
  * @see [TestReactions]
  */
-class ConsoleChannel(override val botApi: BotApi) : BotChannel {
-
-    private val clientId = UUID.randomUUID().toString()
+class ConsoleChannel(override val botApi: BotApi, private val clientId: String = UUID.randomUUID().toString()) : BotChannel {
 
     /**
      * Starts to receive requests from console
      *
      * @param startMessage an optional message that should be printed on startup
      */
     fun run(startMessage: String? = null) {
-        startMessage?.let { process(startMessage) }
+        if (!startMessage.isNullOrBlank()) {
+            println("> $startMessage")
+            process(startMessage)
+        }
+
         while (true) {
             print("> ")
             val input = readLine()
-
-            if (input.isNullOrBlank()) {
-                continue
+            if (!input.isNullOrBlank()) {
+                process(input)
             }
-
-            process(input)
         }
     }
 
     private fun process(input: String) {
-        execute(input).replies.forEach { reply ->
-            print("< ")
-            println(reply)
-        }
-    }
-
-    private fun execute(text: String): TestReactions {
-        val request = QueryBotRequest(clientId, text)
+        val request = QueryBotRequest(clientId, input)
         val reactions = TestReactions()
 
         botApi.process(request, reactions, RequestContext.DEFAULT)
-        return reactions
+
+        render(reactions)
+    }
+
+    private fun render(reactions: TestReactions) {
+        val answer = reactions.executionContext.reactions.joinToString("\n") {
+            render(it).joinToString("\n  ", prefix = "< ")
+        }
+
+        println() // flush conversation logs
+        println(answer)
+    }
+
+    private fun render(reaction: Reaction): List<String> = when (reaction) {
+        is SayReaction -> reaction.text.split("\n")
+        is ImageReaction -> listOf("${reaction.imageUrl} (image)")
+        is AudioReaction -> listOf("${reaction.audioUrl} (audio)")
+        is ButtonsReaction -> listOf(reaction.buttons.joinToString(" ") { "[$it]" })
```

Comment:
What about other reactions like "go", "changeState", "video" and "file"? 
I think we should support all the reactions provided in core.